README.md

```diff
@@ -1,33 +1,34 @@
+For more information about the changes made in jQuery 3.0, see the [upgrade guide](http://jquery.com/upgrade-guide/3.0/) and [blog post](http://blog.jquery.com/FILL-THIS-IN-UPON-FINAL-RELEASE).
 
-In your web page, load this plugin *after* the script for jQuery:
+In your web page, load this plugin *after* the script tag for jQuery, for example:
 
 ```html
-<script src="http://code.jquery.com/jquery-1.9.0.js"></script>
-<script src="http://code.jquery.com/jquery-migrate-1.3.0.js"></script>
+<script src="http://code.jquery.com/jquery-3.0.0.js"></script>
+<script src="http://code.jquery.com/jquery-migrate-3.0.0.js"></script>
 ```
 
-The plugin can be included with versions of jQuery as old as 1.6.4 to identify potential upgrade issues via its JQMIGRATE console warnings.
-However, the plugin is only required for version 1.9.0 or higher to restore deprecated and removed functionality.
+The jQuery Migrate 3.0 plugin should be used only with jQuery 3.x releases. It can identify potential upgrade issues via its JQMIGRATE console warnings.
 
 ## Download
 
 ### Development vs. Production versions
 
-To make it easier for jQuery developers to find and remove deprecated functionality, the development version of the plugin displays warnings on the browser's console. In browsers that don't support the console interface such as IE7, no messages are generated unless you include a debugging library such as [Firebug Lite](https://getfirebug.com/firebuglite) before including the jQuery Migrate plugin. Developers can also inspect the `jQuery.migrateWarnings` array to see what error messages have been generated.
+To make it easier to find and remove deprecated functionality, the development version of the plugin displays warnings on the browser's console. In browsers that don't support the console interface at all times such as IE9, no messages may be generated unless you include a debugging library such as [Firebug Lite](https://getfirebug.com/firebuglite) before including the jQuery Migrate plugin. Developers can also inspect the `jQuery.migrateWarnings` array to see what error messages have been generated.
 
 All warnings generated by this plugin start with the string "JQMIGRATE". A list of the warnings you may see are in [warnings.md](https://github.com/jquery/jquery-migrate/blob/master/warnings.md).
 
 ### Development version
 
-This version provides console warning messages when deprecated and/or removed APIs are used. Use this version during development and debugging, and whenever you are reporting bugs to the jQuery team.
+The development version provides console warning messages when deprecated and/or removed APIs are used. Use this version during development and debugging, and whenever you are reporting bugs to the jQuery team.
 
 **Latest released development version:** This file is hosted on jQuery's CDN, and can be hotlinked if desired.
 [http://code.jquery.com/jquery-migrate-1.3.0.js](http://code.jquery.com/jquery-migrate-1.3.0.js)
```

Comment:
These references to 1.3 will be updated by the release script on our first release.